.github/workflows/ubuntu-lint.yml

```diff
@@ -47,3 +49,28 @@ jobs:
         run: bin/rake check_rvm_integration man:check vendor:check version:check check_rubygems_integration
         if: matrix.ruby.name != 'jruby'
     timeout-minutes: 15
+
+  zizmor:
+    name: zizmor latest via PyPI
+    runs-on: ubuntu-latest
+    permissions:
+      security-events: write
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v4
+        with:
+          persist-credentials: false
+
+      - name: Install the latest version of uv
+        uses: astral-sh/setup-uv@c7f87aa956e4c323abf06d5dec078e358f6b4d04 # v6.0.0
```

Comment:
Should we move the `spell` check to this workflow, add `zizmor` to the requirements file used there, so that we run a consistent version of `zizmor`? I guess we can have an "install python lint deps" job that's a prerequisite for both this and the spelling job.

Reply:
Depending on how adventurous you feel, you could try out https://github.com/zizmorcore/zizmor-action here -- that currently does the `uvx` step under the hood, but I'm planning on removing that in the near future.